Software/Frontend/app/src/main/java/hr/foi/air/baufind/MainActivity.kt

```diff
@@ -25,6 +30,7 @@ class MainActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
 
         setContent {
+            val jobViewModel : JobViewModel = viewModel()
```

Comment:
Bi li ovo trebalo biti tako? Molio bih da čujem ostali što misle.
![](https://www.familyhandyman.com/wp-content/uploads/2018/07/shutterstock_1043376823-smell.jpg)

Reply:
Jel ovo referenca na code smell kolega?

Reply:
Ne kontam sta je problem hahahah